### ISSUE TITLE
REPL & Remove All Files from tool menu stopped working in PyCharm 2020.2

### ISSUE THREAD

**Issue by palmtreefrb** [vlasovskikh/intellij-micropython#117]
In the latest releases microrepl.py is not executing from tool menu. It just opens an empty terminal.
I can manually execute from terminal command line...

frb@ubuntu:~/.local/share/JetBrains/PyCharm2020.2/intellij-micropython/scripts$ python3 microrepl.py /dev/ttyUSB1
Device path /dev/ttyUSB1
Quit: Ctrl+] | Stop program: Ctrl+C | Reset: Ctrl+D
Type 'help()' (without the quotes) then press ENTER.

>>>

**Comment by cr0mbly** [comment]
This is happening to me as well.

**Comment by gbaranski** [comment]
Same issue on MacOS Catalina

**Comment by hlovatt** [comment]
Similar issue MacOS Catalina, PyCharm CE 2020.2, & plugin 1.1.1.

When attempt to open MicroPython REPL I get:

```
L8 n
/Applications/PyCharm CE.app/Contents/plugins/terminal/.zshrc:source:45: no such file or directory: /Users/lov080/Google
```

Is this because `Google Drive` has a space in it? 

Also note strange `L8 n`?

Also:

```
% python3 microrepl.py /dev/tty.usbmodem2067346A344D2 
Traceback (most recent call last):
  File "microrepl.py", line 32, in <module>
    from serial import Serial, SerialException
ModuleNotFoundError: No module named 'serial'
```

**Comment by vlasovskikh** [comment]
@palmtreefrb Thanks for your bug report! I've found a regression in the terminal subsystem of the IntelliJ platform and came up with a workaround. I'll publish the fix soon.

**Comment by palmtreefrb** [comment]
Ok, looks good!!! Thank you...
![Screenshot from 2020-09-20 20-35-051](https://user-images.githubusercontent.com/7799495/93731241-09ae6b00-fb81-11ea-9f36-2ec9ca8989a2.png)

**Comment by vlasovskikh** [comment]
I've released 1.1.2 with the fix.
